src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -679,6 +680,20 @@ private void RepairIndices()
 			_containersForIndexRepair.Clear();
 		}
 
+		private void CleanUpContainers(int startingIndex, int length)
+		{
+			if (ShouldItemsControlManageChildren) return;
```

Comment:
```suggestion
			if (ShouldItemsControlManageChildren)
			{
				return;
			}
```

I known you don't like them, but it's known to produces more robust code (cf. [apple's "gotofail" bug](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/)), but it's also the code styling that was chosen for the whole project :)

Reply:
we can compromise with `if (that) { return; }` in oneline, right?